Assets/_Scripts/Enemy/EnemyProjectile.cs

```diff
@@ -0,0 +1,44 @@
+using System.Collections;
+using System.Collections.Generic;
+using Unity.VisualScripting;
+using UnityEngine;
+
+public class EnemyProjectile : MonoBehaviour
+{
+    Rigidbody2D rb;
+
+    [HideInInspector] public int speed;
+    [HideInInspector] public int offset;
```

Comment:
Why are these two variables hidden in the inspector?

Reply:
We chose to hide these two in the inspector because they have to be public as they are called from another class but we don't want their values changed with in the inspector as they are set within the enemy class.